sources/nl/countrywide.json

```diff
@@ -6,17 +6,22 @@
 		},
 		"country": "nl"
 	},
-	"data": "https://basisregistraties.arcgisonline.nl/arcgis/rest/services/BAG/BAG/MapServer/0",
-	"protocol": "ESRI",
+	"data": "https://data.nlextract.nl/bag/csv/bag-adressen-laatst.csv.zip",
+	"protocol": "http",
+        "compression": "zip",
 	"conform": {
-		"format": "geojson",
+		"format": "csv",
+		"csvsplit": ";",
+		"lon": "lon",
+		"lat": "lat",
 		"number": {
 			"function": "format",
 			"fields": ["huisnummer", "huisletter", "huisnummertoevoeging"],
 			"format": "$1$2-$3"
 		},
-		"street": "openbareruimtenaam",
+		"street": "openbareruimte",
 		"postcode": "postcode",
-		"city": "woonplaatsnaam"
+		"city": "woonplaats",
+		"region": "provincie"
```

Comment:
@stvno I added in the region, do you think that's ok?

Reply:
@andrewharvey yes, provincie means `province` so it falls in the `region` attribute tag